### PR TITLE
Travis CI fixes

### PR DESCRIPTION
**Pull request by amcp**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -256,7 +256,7 @@
                                     <waitAfterLaunch>1</waitAfterLaunch>
                                     <arguments>
                                         <argument>java</argument>
-                                        <argument>-Djava.library.path=dynamodb/DynamoDBLocal_lib</argument>
+                                       <argument>-Djava.library.path=dynamodb/DynamoDBLocal_lib</argument>
```
Comment (amcp) [inline pom.xml]: please add the space back

File: .travis.yml [hunk 2]
```diff
@@ -11,15 +11,15 @@ cache:
...
+    - MODULE='multiple' CATEGORY="com.amazon.titan.testcategory.MultipleItemTests"
+    - MODULE='single' CATEGORY="com.amazon.titan.testcategory.SingleItemTests"
 
 
 matrix:
   # https://docs.travis-ci.com/user/customizing-the-build#Rows-that-are-Allowed-to-Fail
   allow_failures:
-    - env: MODULE='single' CATEGORY="com.amazon.titan.testcategory.MultipleItemTests"
-    - env: MODULE='multiple' CATEGORY="com.amazon.titan.testcategory.SingleItemTests"
+    - env: MODULE='multiple' CATEGORY="com.amazon.titan.testcategory.MultipleItemTests"
```
Comment (amcp) [inline .travis.yml]: I guess the additional split builds will go here. Does the order matter? if not lets not change here

**Comment by amcp** [comment]
Also, please rebase.

File: .travis.yml [hunk 3]
```diff
@@ -31,12 +31,12 @@ branches:
...
 
 script:
-  - mvn install -P integration-tests -Dexclude.category=${CATEGORY} -Dinclude.category="**/*.java"
+  - mvn install -P integration-tests -Dgroups="${CATEGORY}" -Dinclude.category="**/*.java"
   - bash src/test/resources/install-gremlin-server.sh
 
 notifications:
   email:
-    - amcp@amazon.co.jp
+    - johan@nichestreem.com
```
Comment (amcp) [inline .travis.yml]: Feel free to add your self but don't replace me :)

**Comment by F2006** [comment]
@amcp I propose to rather close this pull request. Most of these commits was part of testing builds on Travis CI.

I'll rebase and make the changes as discussed on my side and then we can create the pull request from there. This way we keep the commits clean in this project.

**Comment by amcp** [comment]
@F2006 closing per request

**Pull request closed without merging**